File: programming/lab2/src/main/java/pokemons/Swadloon.java
package pokemons;

import moves.status.GrassWhistle;

public class Swadloon extends Sewaddle {
    public Swadloon(String name, int level) {
        super(name, level);
        this.setStats(55, 63, 90, 50, 80, 42);
        this.addMove(new GrassWhistle());
    }

    public Swadloon() {
        this("Swadloon", 1);
    }
}

File: programming/lab2/app/src/main/kotlin/org/itmo/pokemons/Swadloon.kt
package org.itmo.pokemons

import org.itmo.moves.status.GrassWhistle

open class Swadloon(name: String, level: Int): Sewaddle(name, level) {
    init {
        addMove(GrassWhistle())
        setStats(55.0, 63.0, 90.0, 50.0, 80.0, 42.0)
    }
}